### PR TITLE
Fix Customer Debug Functions in Dev Console Dev Helper (closes #995)

### LINKED ISSUE #995: $dh->debugCustomerByEmail is not working
## Describe the bug

Debugging Customer Info does not work:

## Expected behaviour

Shows Debug array

## Steps to reproduce the issue

```
>>> $dh->debugCustomerByEmail('test@example.com');
PHP Error:  Call to undefined method Magento\Customer\Model\Data\Customer::debug() in phar:///usr/local/bin/n98-magerun2/src/N98/Magento/Command/Developer/DevelopmentHelper.php on line 36
```

## Possible Fix

Reason is, that the CustomRepository returns a Data Model which does not have the debug() function.

This works:

```
>>> $di->get('Magento\Customer\Model\Customer')->loadByEmail('test@example.com')->debug();
```

So we would probably simply replace that code with the workaround.

### PR DESCRIPTION
**Pull request by amenk**
- [x] Pull request against develop branch (if not, just close and create a new one against it)
- [no] README.md reflects changes (if any) [just a fix]
- [?] phar fuctional test (in tests/phar-test.sh)

Fixes #995

### CODE REVIEW

File: src/N98/Magento/Command/Developer/DevelopmentHelper.php [hunk 1]
```diff
@@ -80,7 +80,16 @@ public function getOrderRepository()
...
+        return $this->getCustomerModel()->load($id)->debug();
+    }
+
+    /**
+     * @param $email
+     * @return array
+     */
+    public function debugCustomerByEmail($email)
+    {
+        return $this->getCustomerModel()->loadByEmail($email)->debug();
```
Comment (cmuench) [inline src/N98/Magento/Command/Developer/DevelopmentHelper.php:92]: We need the website_id parameter here if it's a multi store environment.

File: src/N98/Magento/Command/Developer/DevelopmentHelper.php [hunk 2]
```diff
@@ -91,9 +100,13 @@ public function getCustomerRepository()
...
 
-    public function debugCustomerByEmail($email, $websiteId = 0)
+    /**
+     * @return \Magento\Customer\Model\Customer
+     */
+    public function getCustomerModel()
     {
-        return $this->getCustomerRepository()->get($email, $websiteId)->debug();
+        return $this->di->get(\Magento\Customer\Model\Customer::class);
+
```
Comment (cmuench) [inline src/N98/Magento/Command/Developer/DevelopmentHelper.php:109]: This empty line violates the coding standards and let the github action fail.

**Review by cmuench — CHANGES_REQUESTED** [review]
Hi Alex,
thanks for the PR.
Can you fix this two issues of my review?

**Comment by amenk** [comment]
I have a type in my comment :-)

        // Remark: For simpliciy website is not validated and only has an effect, if
        // the configuration customer/account_share/scope is 1

Before I fix it, I want to ask if it is fine like this or if we should add more checks (for the correct website especially)

@cmuench

**Comment by cmuench** [comment]
> I have a typo in my comment :-)
> 
> ```
>     // Remark: For simpliciy website is not validated and only has an effect, if
>     // the configuration customer/account_share/scope is 1
> ```
> 
> Before I fix it, I want to ask if it is fine like this or if we should add more checks (for the correct website especially)
> 
> @cmuench

Good question. If we need to check that we can also use this method here: `\Magento\Customer\Model\Config\Share::isWebsiteScope`

**Comment by amenk** [comment]
@cmuench and how validate the website-id ? Is there something integrated in Magerun or do yo we use core functionality for this?

**Comment by amenk** [comment]
okay, thanks .. looks a bit too much for a dev-helper I would say.

I just fixed the typo for now -- it's a bugfix anyways, and we don't have to over engineer it :-)

**Review by cmuench — APPROVED** [review]

**Pull request merged**